scheduler/ecs/ecs.go

```diff
@@ -597,9 +609,22 @@ func taskDefinitionToProcess(td *ecs.TaskDefinition) (*scheduler.Process, error)
 		Env:         env,
 		CPUShares:   uint(*container.Cpu),
 		MemoryLimit: uint(*container.Memory) * MB,
+		Nproc:       ulimit(td, "nproc"),
```

Comment:
Also, for existing Empire installations, I think there will be a possibility that `container.Ulimits` will be nil, which we'd want to handle.